domains/peace-devops.json

```
@@ -0,0 +1,12 @@
+{
+  "description": "Learning DevOps",
+  "repo": "https://github.com/peacetran2018/devops-docs",
+  "owner": {
+    "username": "peacetran2018",
+    "email": "peacetran2018@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://peacetran2018.github.io"
```

website doesnt really contain anything
also CNAME should not have protocol
```suggestion
    "CNAME": "peacetran2018.github.io"
```